site/content/en/docs/Tutorials/ebpf_tools_in_minikube.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Running eBPF Tools in Minikube"
+linkTitle: "Running eBPF Tools in Minikube"
+weight: 1
+date: 2019-08-15
+description: >
+  Running eBPF Tools in Minikube
+---
+
+## Overview
+
+eBPF tools are performance tools used for observing the Linux kernel.
+These tools can be used to monitor your Kubernetes application in minikube.
+This tutorial will cover how to set up your minikube cluster so that you can run eBPF tools from a Docker container within minikube. 
+
+## Prerequisites
+
+- minikube source code
+- Latest minikube binary
+
+## Tutorial
+
+First, you will need to build the minikube ISO from scratch.
+Instructions for building the ISO can be found [here](../Contributing/iso.md).
+
+_Note: Building the ISO can take at least an hour._
+
+Now, start minikube:
+
+```
+$ minikube start
+```
+
+Mount kernel headers that were built during the ISO build into the minikube VM:
+
+```
+$ cd minikube
+$ minikube mount $(pwd)/out/buildroot/output/build/linux-4.19.94:/lib/modules/4.19.94/build
+```
+
+This command will need to stay running for as long as you want to run BPF tools.
+
+In another terminal window, you can now run [BCC tools](https://github.com/iovisor/bcc) as a Docker container in minikube:
```

Comment:
Would it be better or easier to provide a BCC package for minikube ? https://github.com/iovisor/bcc/blob/master/INSTALL.md

Reply:
For now I think it's easier to run as a container because then we don't have to include extra dependencies in the ISO (python is the first one I can think of off the top of my head)